webpack.config.js

```diff
@@ -9,19 +9,19 @@ const { version } = require("./package.json");
 const OUTPUT_FOLDER = path.join(__dirname, "build");
 const CODEBASE = path.join(__dirname, "source");
 
-const { NODE_ENV, MODE } = process.env;
+const { NODE_ENV, MODE, PUBLIC_URL_ROOT } = process.env;
 const isProd = NODE_ENV === "prod";
 
-const urlPrefix = MODE === "WEBSITE" ? "bitrise_workflow_editor-" : "";
-const publicPath = `/${urlPrefix}${version}/`;
+const urlPrefix = MODE === "WEBSITE" ? PUBLIC_URL_ROOT : "";
+const publicPath = `${urlPrefix}/${version}`;
```

Comment:
hmm I've just checked locally, we need that trailing `/` at the end otherwise it breaks locally :(

Reply:
E2E test does not break unfortunately because the problem only exists on bitrise.yml editor page @simonmarton

Reply:
ack 😢